sphynx/lynxkite-sphynx/networkit_create_graph.go

```diff
@@ -33,6 +68,23 @@ func init() {
 					count("connected_at_start"))
 				defer networkit.DeleteBarabasiAlbertGenerator(g)
 				result = g.Generate()
+			case "StaticDegreeSequenceGenerator":
+				degs := degreeSequence("degrees")
+				defer networkit.DeleteUint64Vector(degs)
+				switch getOpt("algorithm").(string) {
+				case "Chung–Lu":
+					g := networkit.NewChungLuGenerator(degs)
+					defer networkit.DeleteChungLuGenerator(g)
+					result = g.Generate()
+				case "Edge switching Markov chain":
+					g := networkit.NewEdgeSwitchingMarkovChainGenerator(degs)
+					defer networkit.DeleteEdgeSwitchingMarkovChainGenerator(g)
+					result = g.Generate()
+				case "Haveli–Hakimi":
+					g := networkit.NewHavelHakimiGenerator(degs)
+					defer networkit.DeleteHavelHakimiGenerator(g)
+					result = g.Generate()
+				}
```

Comment:
Shouldn't we also provide these in a form that creates edges only for existing vertices, using a vertex attribute for the requested degree?

Reply:
> Shouldn't we also provide these in a form that creates edges only for existing vertices, using a vertex attribute for the requested degree?

Ah, that would be pretty cool! Maybe in a follow-up.